pod/perlgov.pod

```diff
@@ -427,7 +433,9 @@ anonymously.  After voting is complete, candidates are ranked using the
 Condorcet Internet Voting Service's proportional representation mode.  If a tie
 occurs, it may be resolved by mutual agreement among the tied candidates, or
 else the tie will be resolved through random selection by the Vote
-Administrator.
+Administrator.  Unless there are more candidates than open seats, no vote is
```

Comment:
As with code, I find "unless" harder to mentally parse than an "if".  Also, the rest of the document uses the term "vacancy", not "open seats" so I suggest that be changed for consistently.  I have a mild preference that we call this out either at the top of item 2 or as a new item 2 so that this check appears between "nomination" and "ranking" in the document in the same way it will happen in the actual process.